code/game/machinery/cryopod.dm

```diff
@@ -551,3 +552,40 @@ GLOBAL_LIST_INIT(frozen_items, list(SQUAD_MARINE_1 = list(), SQUAD_MARINE_2 = li
 		return
 
 	move_inside(target)
+
+
+/obj/structure/machinery/cryopod/tutorial
+	silent_exit = TRUE
+
+/obj/structure/machinery/cryopod/tutorial/process()
+	return
+
+/obj/structure/machinery/cryopod/tutorial/go_in_cryopod(mob/mob, silent = FALSE, del_them = TRUE)
+	if(occupant)
+		return
+	mob.forceMove(src)
+	occupant = mob
+	icon_state = "body_scanner_closed"
+	set_light(2)
+	time_entered = world.time
+	if(del_them)
+		despawn_occupant()
+
+/obj/structure/machinery/cryopod/tutorial/despawn_occupant()
+	SSminimaps.remove_marker(occupant)
+
+	if(ishuman(occupant))
+		var/mob/living/carbon/human/man = occupant
+		man.species.handle_cryo(man)
+
+	icon_state = "body_scanner_open"
+	set_light(0)
+
+
+	var/mob/new_player/new_player = new
+
+	if(!occupant.mind)
+		occupant.mind_initialize()
+
+	occupant.mind.transfer_to(new_player)
+	QDEL_NULL(occupant)
```

Comment:
This causes a delete loop runtime

cryopod destroys occupant -> end_tutorial picks up deletion and deletes tutorial -> tutorial destroys turf reservation -> clearing the reservation destroys occupant

Reply:
This fixes it from what I can tell